aci/compose.go

```diff
@@ -86,11 +87,36 @@ func (cs *aciComposeService) Copy(ctx context.Context, project *types.Project, o
 }
 
 func (cs *aciComposeService) Up(ctx context.Context, project *types.Project, options api.UpOptions) error {
+	if err := checkUnsupportedUpOptions(options); err != nil {
+		return err
+	}
 	return progress.Run(ctx, func(ctx context.Context) error {
 		return cs.up(ctx, project)
 	})
 }
 
+func checkUnsupportedUpOptions(o api.UpOptions) error {
+	var errs *multierror.Error
+	checks := []struct {
+		toCheck, expected interface{}
+		option            string
+	}{
+		{o.Start.CascadeStop, false, "abort-on-container-exit"},
+		{o.Create.RecreateDependencies, "", "always-recreate-deps"},
+		{len(o.Start.AttachTo), 0, "attach-dependencies"},
+		{len(o.Start.ExitCodeFrom), 0, "exit-code-from"},
+		{o.Create.Recreate, "", "force-recreate"},
+		{o.Create.QuietPull, false, "quiet-pull"},
+		{o.Create.RemoveOrphans, false, "remove-orphans"},
+		{o.Create.Inherit, true, "renew-anon-volumes"},
+		{o.Create.Timeout, nil, "timeout"},
+	}
+	for _, c := range checks {
+		errs = utils.CheckUnsupported(errs, c.toCheck, c.expected, "up", c.option)
```

Comment:
This should just be
```suggestion
		if err := utils.CheckUnsupported(c.toCheck, c.expected, "up", c.option); err != nil {
			errs = multierror.Append(errs, err)
		}
```

Reply:
The problem with this approach is that by taking this `if` outside of the function I will have to repeat it all over the code. And that's the very same `if`. Leaving this mutualised in the `CheckUnsupported` avoids this repetition.

Reply:
Yes, OK, good point!